src/BunqLib.php

```diff
@@ -245,6 +245,30 @@ public function makePayment(
         )->getValue();
     }
 
+    /**
+     * @param string $amount
+     * @param string $recipient
+     * @param string $description
+     * @param string $name
+     * @param MonetaryAccountBank $monetaryAccount
+     *
+     * @return int
+     */
+    public function makePaymentIBAN(
+        string $amount,
+        string $recipient,
+        string $description,
+	string $name,
```

Comment:
indentation is not correct, pleas indent correctly.

Reply:
identation fixed.